RELEASE_CHANGELOG.md

```diff
@@ -264,26 +264,3 @@ This version based on finschia-sdk v0.46.0-rc2
 
 ### Build, CI
 * (build) [\#25](https://github.com/Finschia/finschia/pull/25) Fix localnet-start
-
-## [gaia v4.0.4] - 2021-03-15
-Initial lbm is based on the tendermint v0.34.9+, cosmos-sdk v0.42.0+, gaia v4.0.4
-
-* (tendermint) [v0.34.9](https://github.com/tendermint/tendermint/releases/tag/v0.34.9).
-* (cosmos-sdk) [v0.42.0](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.42.0).
-* (gaia) [v4.0.4](https://github.com/cosmos/gaia/releases/tag/v4.0.4).
-
-Please refer [CHANGELOG_OF_GAIA_v4.0.4](https://github.com/cosmos/gaia/blob/v4.0.4/CHANGELOG.md)
-
-
-<!-- Release links -->
-[v3.0.0]: https://github.com/Finschia/finschia/releases/tag/v3.0.0
-[v2.0.0]: https://github.com/Finschia/finschia/releases/tag/v2.0.0
-[v1.0.0]: https://github.com/Finschia/finschia/releases/tag/v1.0.0
-[v0.7.0]: https://github.com/Finschia/finschia/releases/tag/v0.7.0
-[v0.6.0]: https://github.com/Finschia/finschia/releases/tag/v0.6.0
-[v0.5.0]: https://github.com/Finschia/finschia/releases/tag/v0.5.0
-[v0.4.0]: https://github.com/Finschia/finschia/releases/tag/v0.4.0
-[v0.3.0]: https://github.com/Finschia/finschia/releases/tag/v0.3.0
-[v0.2.0]: https://github.com/Finschia/finschia/releases/tag/v0.2.0
-[v0.1.0]: https://github.com/Finschia/finschia/releases/tag/v0.1.0
-[gaia v4.0.4]: https://github.com/cosmos/gaia/releases/tag/v4.0.4
```

Comment:
Why did you remove this realease links and gaia update changelog?

Reply:
Why do we need to leave Gaia release? becasue it's Finschia's release changelog. But I will revert our release link.

Reply:
Because finschia was forked from gaia 4.0.4.